base/applications/rapps/include/rosui.h

```diff
@@ -302,54 +300,78 @@ class CUiBox
         *newRect = currentRect;
     }
 
-
 public:
+    //virtual ~CUiBox() {} // Needed for current CUiCollection implementation.
+
+    // Override in subclass
     virtual VOID ComputeMinimalSize(SIZE* size)
     {
-        // Override in subclass
         size->cx = max(size->cx, 0);
         size->cy = min(size->cy, 0);
-    };
+    }
 
+    // Override in subclass
     virtual VOID ComputeContentBounds(RECT* rect)
     {
-        // Override in subclass
-    };
+    }
 
-    virtual DWORD_PTR CountSizableChildren()
+    // Override in subclass
+    virtual INT CountSizableChildren()
```

Comment:
I want to uniformly change the returned type of this function (here and in the derived classes and where it gets used) to `UINT`, as it is obvious a number of objects won't be negative.